docs/about.md

```diff
@@ -55,16 +55,51 @@ Contributor [2021 January - ]<br/>
 
 <!-- ------------------------------------------------------------------------------------------------------ -->
 
-### [Gokul Rajiv](https://github.com/gok99)
+### [Zhou Jiahao](https://github.com/zhoukerrr)
 **Role**: Contributor [2022 January - ]<br/>
 
 <!-- ------------------------------------------------------------------------------------------------------ -->
 
-### [Zhou Jiahao](https://github.com/zhoukerrr)
+### [Charisma Kausar](https://www.github.com/ckcherry23)
+**Role**: Contributor [2024 January - ]<br/>
```

Comment:
```suggestion
**Role**: Committer [2024 January - ]<br/>
Contributor [2022 December - ]<br/>
```

Reply:
Would it be safe to assume that all current mentors have the same title? (except for maybe @gok99 who was already on the list?)

Reply:
Yes, we should have the same title. Even @gok99 would be a Committer now. But his contributor start date should be a year earlier.

Reply:
I see, I have updated the commit now, it should reflect the correct roles and start dates now!